sopel/modules/reload.py

```diff
@@ -30,29 +30,36 @@ def _load(bot, plugin):
         raise
 
 
-@sopel.module.nickname_commands("reload")
-@sopel.module.priority("low")
-@sopel.module.thread(False)
-@sopel.module.require_admin
+@plugin.nickname_command("reload")
+@plugin.priority("low")
+@plugin.thread(False)
+@plugin.require_admin
+@plugin.output_prefix(PLUGIN_OUTPUT_PREFIX)
 def f_reload(bot, trigger):
     """Reloads a plugin (for use by admins only)."""
     name = trigger.group(2)
 
     if not name or name == '*' or name.upper() == 'ALL THE THINGS':
         bot.reload_plugins()
-        return bot.reply('done')
+        bot.say('done')
+        return
 
     if not bot.has_plugin(name):
-        return bot.reply('"%s" not loaded, try the `load` command' % name)
+        bot.reply('"%s" not loaded, try the `load` command' % name)
```

Comment:
Another case of "let's fix this because it's already touched anyway".

```suggestion
        bot.reply('"%s" not loaded; try the `load` command.' % name)
```

Could use `help_prefix` too, I guess. Up to you.